bot/utility/utility.py

```diff
@@ -212,6 +215,29 @@ async def welcome_message(self, user: discord.Member):
                                                               "__gemeinsam__ schaffen wir das!** :muscle:")
         await user.send(content=content, embed=embed)
 
+    @commands.command(name='translate')
+    @command_log
+    async def translate_msg(self, ctx: commands.Context, *, msg_id: int):
+        """Command Handler for the `translate` command.
+
+        Translates the message with the provided id to english and sends the translation as a dm to the user
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            msg_id (int): The id of the message that should be translated
+
+        """
+        message = await ctx.fetch_message(msg_id)
+        if constants.DEEPL_API_KEY == "undefined":
```

Comment:
the word "undefined" in constants.py has a uppercase U, whereas this one has lowercase, so the comparison will always fail.

Reply:
fixed